src/service/write-off-service.ts

```diff
@@ -142,20 +149,22 @@ export default class WriteOffService {
       toId: user.id,
       description: 'Write off',
       fromId: null,
-    }, manager);
+    });
 
     const highVatGroup = await WriteOffService.getHighVATGroup();
     writeOff.transfer = transfer;
     transfer.writeOff = writeOff;
     transfer.vat = highVatGroup;
 
-    await manager.save(transfer);
-    await manager.save(writeOff);
+    await this.manager.save(transfer);
+    await this.manager.save(writeOff);
     return WriteOffService.asWriteOffResponse(writeOff);
   }
 
-  public static async createWriteOffAndCloseUser(user: User): Promise<WriteOffResponse> {
-    const writeOff = await wrapInManager(WriteOffService.createWriteOff)(user);
+  // TODO: This should be a transaction
+  //   wait for BalanceService to be refactored
```

Comment:
Wait, how/why does `BalanceService` need to be refactored? Is there a GH issue for that?

Reply:
This is just the general non-static class refactor, to give it its own enitity manager.